osbrain/tests/test_proxy.py

```diff
@@ -119,6 +122,42 @@ def ping(self):
         InitTimeoutProxy('foo', timeout=1.)
 
 
+def test_agent_proxy_wait_running(nsproxy):
+    """
+    A proxy initialized with running=True should wait for a number of seconds
+    until the agent is running
```

Comment:
I think this docstring is outdated. :wink:

Reply:
Definitely :smile: